docs/calamari/CommunityCollatorsProgram.md

```diff
@@ -0,0 +1,284 @@
+# 🧩  Community Collators Program
+
+## Requirements
+
+1. Hardware Requirement
+
+    a. If running on your own machine, hardware requirement, internet bandwidth
+    
+    ```jsx
+    cpu:8 cores
+    memory: 32 GB of ram
+    disk space: 100 GB of disk space
+    recommended region: EU or US
+    ```
+    
+    b. If running on AWS, instance type and the location guide
+    
+    ```jsx
+    AWS instance: Ubuntu 20.04 (ami-id = ami-04505e74c0741db8d)
+    disk space: r5ad.xlarge
+    recommended region: ca-central, eu-west, eu-central, eu-north, us-east
+    ```
+    
+    NOTE: 
+    
+    - A calamari collator utilizes up to 27gb of ram on a 32gb system.
+    - A calamari collator utilizes up to 25% of cpu on a 24 core system.
+    - A reliable internet connection as well as power supply which should remain uninterrupted to keep local machine **permanently online.**
+2. Bonding requirement: `400_000`KMA (To be changed) at least. Make sure your account has more than `400_000`KMA in your free balance. 
```

Comment:
Maybe point people to the candidacyBond() getter on polkadotjs to find out the current requirement as it might rise in the future

![image](https://user-images.githubusercontent.com/4237655/151414136-3b88ed6a-16c0-4e71-a320-ec70ff8796e9.png)